tools/integration_tests/list_large_dir/list_large_dir_test.go

```diff
@@ -24,6 +24,9 @@ import (
 	"github.com/googlecloudplatform/gcsfuse/v2/tools/integration_tests/util/setup"
 )
 
+const DirectoryForListLargeFileTests1 = "directoryForListLargeFileTests1"
```

Comment:
IIUC, only the package is running the tests in prallel to other packages. Couldn't we use just 1 test directory?